tests/pyret/tests/test-equality.arr

```diff
@@ -24,13 +24,23 @@ check "numbers":
   roughly-equal(3, 3) is true
   roughly-equal(3, 3.000003) is true
   roughly-equal-now(3, 3.000003) is true
-  roughly-equal-now(3, 3.000004) is false
```

Comment:
Should this be factored out into a separate PR?

Reply:
_maaaaaaaaaaaaaybe_, but I included it here because getting any sort of roughly-equals behavior on matrices _really needs_ a better notion of "roughly equal to zero": you simply can't say something is "roughly the identity matrix" without having a more tolerant notion of "roughly equal to zero".